bridge/client/app/_views/ktb-environment-view/ktb-stage-overview/ktb-stage-overview.component.ts

```diff
@@ -63,6 +60,7 @@ export class KtbStageOverviewComponent implements AfterContentInit, OnDestroy {
       .pipe(takeUntil(this.unsubscribe$))
       .subscribe(([stageName, filterType, project]) => {
         const stage = project.stages.find((s) => s.stageName === stageName);
+        this.setFilter(project, true);
```

Comment:
Actually, this should be called on project change and should not depend on the other observables. Currently, the service filter is not set when initially loaded. With this change, `setFilter` is also called if `selectedStageName$` or `paramFilterType$` changes

Reply:
Yes, that's true. Should I change it back to using `tap` on the `project$` observable or is there a better solution?

Reply:
We could split it into two observables. We have two subscriptions for `project$` I guess. We have some approaches here:
1. The view (HTML-part) that subscribes to it shall contain the `tap` and the one in `combineLatest` shall not.
2. The view and the `combineLatest` takes the `project$` without the tap and we introduce another one, where we subscribe directly to it.

@heinzburgstaller what would you say?

Reply:
I would manually subscribe to `project$` in the component and only call `setFilter` there. Therefore you have to manually unsubscribe, but it looks cleaner.